tools/hardware_provisioning.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
-USE_RESERVED=${USE_RESERVED:-false}
 DEPLOY_NAME=${DEPLOY_NAME:-cnftestbed}
-PACKET_FACILITY=${PACKET_FACILITY:-sjc1}
-PACKET_OS=${PACKET_OS:-ubuntu_16_04}
-MASTER_COUNT=${MASTER_COUNT:-1}
-MASTER_PLAN=${MASTER_PLAN:-m2.xlarge.x86}
-WORKER_COUNT=${WORKER_COUNT:-1}
-WORKER_PLAN=${WORKER_PLAN:-n2.xlarge.x86}
+STATE_FILE=${STATE_FILE:-$(pwd)/data/$DEPLOY_NAME/terraform.tfstate}
+NODE_FILE=${NODE_FILE:-$(pwd)/data/$DEPLOY_NAME/kubernetes.env}
+USE_RESERVED=${USE_RESERVED:-false}
+
+NODE_GROUP_ONE_NAME=${NODE_GROUP_ONE_NAME:-$DEPLOY_NAME-master}
+NODE_GROUP_TWO_NAME=${NODE_GROUP_TWO_NAME:-$DEPLOY_NAME-worker}
+NODE_GROUP_ONE_COUNT=${NODE_GROUP_ONE_COUNT:-1}
+NODE_GROUP_TWO_COUNT=${NODE_GROUP_TWO_COUNT:-1}
+FACILITY=${FACILITY:-sjc1}
+NODE_GROUP_ONE_DEVICE_PLAN=${NODE_GROUP_ONE_DEVICE_PLAN:-m2.xlarge.x86}
+NODE_GROUP_TWO_DEVICE_PLAN=${NODE_GROUP_TWO_DEVICE_PLAN:-n2.xlarge.x86}
+OPERATING_SYSTEM=${OPERATING_SYSTEM:-ubuntu_16_04}
+
```

Comment:
hardware-provisioning.env.example needs to be updated to match these new variable names

Reply:
Resolved by:
https://github.com/cncf/cnf-testbed/commit/f9b8c75a065a96853a4d4f2db54ee30687976f3e